pkg/search/proxy/store/multi_cluster_cache.go

```diff
@@ -322,6 +353,20 @@ func (c *MultiClusterCache) List(ctx context.Context, gvr schema.GroupVersionRes
 	return resultObject, nil
 }
 
+func watchUseProtobuf(ctx context.Context) bool {
+	acceptValues := ctx.Value("Accept")
+	if acceptValues != nil {
+		avs := acceptValues.([]string)
+		for _, v := range avs {
+			vs := strings.Split(v, ",")
+			if vs[0] == "application/vnd.kubernetes.protobuf" {
+				return true
```

Comment:
There seem to be some default conventions here. Can you explain why only the element at index 0 is being checked?

Reply:
becase this is first accept header.